src/components/createActivityForm/CreateActivityForm.spec.js

```diff
@@ -0,0 +1,65 @@
+import { render, screen } from '@testing-library/react'
+import CreateActivityForm from './CreateActivityForm'
+
+describe('CreateActivityForm', () => {
+  it('has ten required input fields', () => {
```

Comment:
After reading 'has ten required input fields', I thought you would also test `toHaveLength(10)` :-) But you are already testing every input field in your form, so I think that this is not necessary here anymore.